_posts/2023-04-14-h2oai.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "The Return of the H2oai Benchmark"
+author: Tom Ebergen
+excerpt_separator: <!--more-->
+---
+
+*TL;DR: We re-ran the H2O.ai db-benchmarks with up-to-date libraries and DuckDB is consistently a top performer.*
+
+
+[Skip directly to the results](#results)
+
+The H2O.ai [DB benchmark](https://h2oai.github.io/db-benchmark/) is a well-known benchmark in the data analytics and R community. The benchmark measures the groupby and join performance of various analytical tools like data.table, polars, dplyr, clickhouse, duckdb and more. Since July 2nd 2021, the benchmark has been dormant, with no result updates or maintenance. Many of the analytical systems measured in the benchmark have since undergone substantial improvements, leaving many of the maintainers curious as to where their analytical tool ranks on the benchmark.
+
+DuckDB has decided to give the H2O.ai benchmark new life and maintain it for the foreseeable future. One reason for DuckDB Labs' decision to maintain the benchmark is that DuckDB has had 4 new minor releases since the most recent published results on July 2nd, 2021. After managing to run parts of the benchmark on a r3-8xlarge AWS box, DuckDB ranked as a top performer on the benchmark. Additionally, DuckDB Labs wants to demonstrate DuckDB's commitment to performance by consistently comparing ourselves with other analytical systems. While DuckDB delivers differentiated ease of use, raw performance and scalability are critically important for solving tough problems fast. Plus, just like many of our fellow data folks, we have a need for speed. Therefore, the decision was made to fork the benchmark, modernize underlying dependencies and run the benchmark on the latest versions of the included systems. You can find the repository [here](https://github.com/duckdb/h2oai-db-benchmark). 
```

Comment:
We had 10 releases since v0.2.7 - we changed our release numbering scheme mid-way (v0.2.7, v0.2.8, v0.2.9, v0.3.0, v0.3.1, v0.3.2, v0.3.3, v0.4.0, v0.5.0, v0.6.0, v0.7.0)

Reply:
ah, I thought it was always Major.Minor.Bugfix. I'll change it to "10 new minor releases"